test/ReactViews/FeatureInfoSectionSpec.tsx

```diff
@@ -1312,6 +1318,34 @@ describe("FeatureInfoSection", function () {
       expect(findWithText(result, "DEF").length).toEqual(1);
     });
   });
+
+  describe("feature info panel buttons", function () {
+    it("renders buttons added using FeatureInfoPanel.addFeatureButton", function () {
+      FeatureInfoPanel.addFeatureButton(viewState, ({ feature, item }) => {
```

Comment:
`item` is not being used. Maybe add test case for it?

Reply:
Good point. I have changed the code to assert the type of item passed through.

Reply:
Where will `item` be used? Or you have not decided yet?

Reply:
So the real purpose of the `item` parameter is if a plugin wants to restrict the button only for certain item types. For eg, scene-editor wants to show a button only for Gltf items it will return the button configuration only if `GltfMixin.isMixedInto(item)`. 

I have [updated the test](https://github.com/TerriaJS/terriajs/blob/d7c08c8f0a92784c4994fd4d1bfdab3b9a0d92a1/test/ReactViews/FeatureInfoSectionSpec.tsx#L1325-L1327) here by inserting a similar assertion.